src/classlibnative/bcltype/system.cpp

```diff
@@ -73,9 +73,11 @@ FCIMPLEND;
 
 
 
-#ifndef FEATURE_CORECLR
 INT64 QCALLTYPE SystemNative::GetWorkingSet()
 {
+#if PLATFORM_UNIX
+    return 0; // TODO: Implement in PAL
```

Comment:
Sounds like I should close this PR and start over with one that follows this new direction?

Alternatively I could get this fixed up and merged, then get the contract and tests in place in CoreFx, and then go back and fix up the implementation following this revised direction, assuming it works.

Preference?